tests/templates/included.html

```diff
@@ -0,0 +1,8 @@
+{% load partials %}
+INCLUDED TEMPLATE START
+{% partialdef included-partial %}
+THIS IS CONTENT FROM THE INCLUDED PARTIAL
+{% endpartialdef %}
+
+Now using the partial: {% partial included-partial %}
+INCLUDED TEMPLATE END
```

Comment:
Please add a final newline here. (And perhaps configure your editor to always add them.)

Reply:
added